docs/api.yaml

```diff
@@ -58,3 +82,17 @@ components:
           items:
             type: string
             example: "In ancient Egypt, killing a cat was a crime punishable by death."
+    error: 
+      type: object
+      properties:
+        code:
+          type: integer
+          format: int32
+          example: 400
+        message:
+          type: string
+          example: "Invalid language, valid languages are 'eng', 'ukr', 'rus'"
```

Comment:
With the changes from my PR, it will look like this: 
<img width="807" alt="image" src="https://user-images.githubusercontent.com/45897337/201740177-eb72f217-fdeb-4cf3-8e74-72b914f846d1.png">

Reply:
I totally forgot I closed the PR before I sent it. 

Thank you so much for this contribution, literally completed it hours after the issue was open. 
While writing this documentation line actually inspired me to make the issue: https://github.com/wh-iterabb-it/meowfacts/issues/216



:clap: :+1:  again! great job, and thanks for contributing!